LondonBibleSociety/BSLet175.xml

```diff
@@ -0,0 +1,81 @@
+   <teiHeader xml:base="https://betamasaheft.eu/">
+      <fileDesc>
+         <titleStmt>
+            <title>Prayers</title>
+            <editor key="JK"/>
+            <editor key="AB" role="generalEditor"/>
+            <funder>Akademie der Wissenschaften in Hamburg</funder>
+         </titleStmt>
+         <editionStmt><p>Cataloguing funded by the AHRC-DFG Project
+            'Demarginalizing medieval Africa: Images, texts, and identity
+            in early Solomonic Ethiopia (1270-1527)', grant ref.
+            nos. AH/V002910/1 - 448410109</p></editionStmt>
+         <publicationStmt>
+            <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+            <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale Forschungsumgebung / Beta maṣāḥǝft</publisher>
+            <pubPlace>Hamburg</pubPlace>
+            <availability>
+               <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                  licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+            </availability>
+         </publicationStmt>
+         <sourceDesc>
+            <msDesc xml:id="ms">
+                  <msIdentifier>
+                     <repository ref="INS0306BSL"/>
+                     <idno>BFBS Mss 175</idno>
+                     <altIdentifier>
+                        <idno>BSMS 175</idno>
+                     </altIdentifier>
+                     <altIdentifier>
+                        <idno>Cowley XXVII</idno>
+                     </altIdentifier>
+                  </msIdentifier>
+                  <history>
+                  </history>
+                  <additional>
+                     <adminInfo>
+                        <recordHist>
+                           <source>
+                              <listBibl type="catalogue">
+                                 <bibl>
+                                    <ptr target="bm:Cowley1982BBS"/>
+                                    <citedRange unit="page">85-87</citedRange>
+                                 </bibl>
+                              </listBibl>                  
+                           </source>
+                        </recordHist>
+                        </adminInfo>
+              
+                  </additional>
+            </msDesc>
+         </sourceDesc>
+      </fileDesc>
+      <encodingDesc>
+         <projectDesc>
+            <p>Encoded according to TEI P5 Guidelines.</p>
+         </projectDesc>
+         <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="https://raw.githubusercontent.com/BetaMasaheft/Documentation/master/prefixDef.xml">
+            <xi:fallback>
+               <p>Definitions of prefixes used.</p>
+            </xi:fallback>
+         </xi:include>
+      </encodingDesc>
+      <profileDesc>
+         <langUsage><language ident="en">English</language></langUsage>
```

Comment:
add also Geez?

Reply:
It is not technically necessary, because they language has not been used in the record. Here, all languages used in the record are defined. But since we assume that the manuscript is written in Geez and will be described later, it won't be wrong (adding languages here that are not used in the manuscript or record is confusing the search results.)

Reply:
Ah, good to know! :)